source/core/src/main/com/csse3200/game/entities/factories/EnemyFactory.java

```diff
@@ -260,6 +265,59 @@ public static Entity createCombatBossNPC() {
     PhysicsUtils.setScaledCollider(npc, 0.9f, 0.4f);
     return npc;
   }
+
+
+  /**
+   * Creates chicken enemy as NPC entity for static combat
+   * */
+  public static Entity createChickenCombatEnemy() {
+    Entity chickenEnemy = createCombatBossNPC();
+    BaseEnemyEntityConfig config = configs.chicken;
+    chickenEnemy.setEnemyType(Entity.EnemyType.CHICKEN);
+
+    chickenEnemy
+            .addComponent(new TextureRenderComponent("images/chicken_idle.png"))
+            .addComponent(new CombatStatsComponent(config.getHealth(), 100, 100, 100, 100, 100, false));
```

Comment:
shouldnt these be the combat stats of the enemy (they've been defined/ can be defined in the config file)